source/api/commands/route.md

```diff
@@ -191,38 +191,14 @@ cy.route('POST', '**/comments', commentsResponse)
 
 Any request that matches the `method` and `url` of a route will be responded to based on the configuration of that route.
 
-If a request doesn't match any route, {% urlHash "it will automatically receive a 404" Notes %}. For example, given we have the following routes:
-
-```javascript
-cy.server()
-cy.route('**/users', [
-  {id: 19, name: 'Laura'},
-  {id: 20, name: 'Jamie'}
-])
-cy.route('POST', '**/messages', {id: 123, message: 'Hi There!'})
-cy.get('form').submit()
-```
-
-```javascript
-// Application Code
-// when our form is submitted
-$('form').submit(() => {
-  // send an AJAX to: GET /users
-  $.get('https://localhost:7777/users' )
-
-  // send an AJAX to: POST /messages
-  $.post('https://localhost:7777/messages', {some: 'data'})
-
-  // send an AJAX to: GET /updates
-  $.get('https://localhost:7777/updates')
-})
-```
+{% note info %}
+If a request doesn't match any route then the behavior depends on the value of the `force404` option on the {% url `cy.server()` server %}:
 
-**The above application code will issue 3 AJAX requests:**
+* if `force404` is `false` (the default) then the request will {% url 'pass through to the server' network-requests#Don’t-Stub-Responses %}.
+* if `force404` is `true` then the response {% urlHash "will be a 404" Notes %}.
 
-1. The `GET /users` will match our 1st route and respond with a `200` status code and the array of users.
-2. The `POST /messages` will match our 2nd route and respond with a `200` status code with the message object.
-3. The `GET /updates` did not match any routes and its response automatically sent back a `404` status code with an empty response body.
+You can {% url 'read more about this behavior here.' server#Options %}
+{% endnote %}
```

Comment:
> Right, so this is rather a large edit, possibly the largest I've made so far. Do steer me Jennifer.

I've removed this example in favour of a short block that directs the reader to the documentation about `force404` on the `server` page. It's a good example but it essentially duplicates a much shorter example on the `server` page. I've attempted to preserve the thrust of this section while also directing the reader to the `server` page because the `force404` option is applied at the level of the `server` rather than the `route`.

Yeah? 🤔

Reply:
This is fine and a good use of a 'note' as it is outside of the flow of the content.